src/imitation/scripts/common/reward.py

```diff
@@ -47,6 +47,8 @@ def normalize_output_running():
 @reward_ingredient.named_config
 def reward_ensemble():
     net_cls = reward_nets.RewardEnsemble  # noqa: F841
```

Comment:
For multi-line configs we normally drop `# noqa: F841` and just add a `locals()` line to the end of the function which also quietens flake8. Hacky, admittedly, but probably still less bad than peppering every line with noqa.